src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -104,13 +104,13 @@ const styles = StyleSheet.create({
   headerDomainName: {
     fontSize: 14,
     fontFamily: fonts.inter[600],
-    color: COLORS.black
+    color: '#000000'
   },
   headerDomainDate: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: COLORS.blackgrey,
+    color: '#828282',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, hardcoded color values are found here as well. Consider using the `COLORS` object for color definitions.
```diff
-    color: '#000000'
...
-    color: '#828282',
+    color: COLORS.black
...
+    color: COLORS.blackgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->